.bin/create-rust-webpack.js

```diff
@@ -18,6 +29,11 @@ function run(cmd, args, opts) {
   }
 }
 
+if (!validateInstallation("git help -g")) {
```

Comment:
Can we rename the function too 👍 

```suggestion
if (!IsPresent("git --version")) {
```